client/my-sites/theme/main.jsx

```diff
@@ -644,6 +643,7 @@ export default connect(
 	( state, { id } ) => {
 		const selectedSite = getSelectedSite( state );
 		const siteSlug = selectedSite ? getSiteSlug( state, selectedSite.ID ) : '';
+		const isJetpack = selectedSite && isJetpackSite( state, selectedSite.ID );
```

Comment:
We don't need the intermediate `const` here because the value is not used in this func. We can add it directly to the returned object below. Also, we should add `isJetpack` to the `propTypes` array at the start of the component.